starter-code/scripts/controllers/aboutController.js

```diff
@@ -4,7 +4,10 @@
   var aboutController = {};
 
   aboutController.reveal = function() {
-    /* TODO: Use your DOM skills to reveal only the about section! */
+    /* TODO: DONE: Use your DOM skills to reveal only the about section! */
+    $('#articles').hide();
+    $('#blog-stats').hide();
+    $('#about').fadeIn(1000);
```

Comment:
This works fine, but you might consider using a class to hide all "pages" and then fade in the appropriate one. The class .tab-content would work well for this. Then, if you had many more pages, you wouldn't have to worry about listing them all out within every reveal method (the same goes for the other controllers' reveal methods :)).